styles.css

```diff
@@ -214,51 +216,55 @@ label{
 }
 
 #robot{
-  vertical-align:middle;
   width:1.5em;
   height:1.5em; 
   margin-right:0.8em;
-  outline:0.15em solid rgb(211, 210, 210);
-  border-radius: 0.1em;
+  outline:0.2em solid rgb(221, 220, 220);
+  border-radius: 15%;
+  cursor: pointer;
+  vertical-align:middle;
```

Comment:
Do you want to center checkbox vertically? Use `align-items:center;` on the `.robot` and remove `padding-top: 2em;` from `.robot` instead